models/iosxr/bgp_global/iosxr_bgp_global.yaml

```diff
@@ -0,0 +1,826 @@
+module: iosxr_bgp_global
+short_description: Manages BGP global resource module.
+description:
+- This module configures and manages the attributes of BGP global on Cisco IOS-XR platforms.
+version_added: 1.0.0
+author: Ashwini Mhatre (amhatre)
+notes:
+- Tested against Cisco IOS-XR 6.1.3.
+- This module works with connection C(network_cli).
+options:
+    config:
+      description: A list of configurations for BGP global.
+      type: dict
+      suboptions:
+        as_number:
```

Comment:
IMO, we should be standardizing the keys based on OpenConfig.
https://github.com/openconfig/public/blob/master/release/models/bgp/openconfig-bgp-global.yang#L103

Reply:
Ack, made changes